core/src/test/java/com/venia/core/models/commerce/MyProductTeaserImplTest.java

```diff
@@ -42,7 +42,7 @@
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 import org.mockito.internal.util.reflection.FieldReader;
-import org.mockito.internal.util.reflection.FieldSetter;
```

Comment:
I wonder why cannot we use FieldSetter here but we can still use FieldReader ?  Tests frameworks provide methods for this which is preferable and would be a better example over providing our methods based on Java reflection.

Reply:
Have removed internal mockito package as it is not the standard way based on my understanding. In place have used org.apache.commons.lang3.reflect.FieldUtils package. Let me know if there are any concerns over this. Thanks!